Tribler/Core/Libtorrent/LibtorrentDownloadImpl.py

```diff
@@ -274,10 +291,17 @@ def network_create_engine_wrapper(self, pstate, initialdlstatus=None):
 
                 atp["ti"] = torrentinfo
                 has_resume_data = resume_data and isinstance(resume_data, dict)
+
+                if has_resume_data is not None:
```

Comment:
In line 271, `resume_data` might be None. If a `None` is operated with True/False, it will be `None`. In this case, `has_resume_data` is `None`. 

In short, if we have `resume_data` but the type is not `dict`, then we tried to properly load it.